server/models/UserProfile.js

```diff
@@ -0,0 +1,36 @@
+const mongoose = require('mongoose')
+const reqString = {
+  type: 'string',
+  required: true
+}
+
+const noReqString = {
+  type: 'string',
+  required: false
+}
+
+const userProfile = new mongoose.Schema(
+  {
+    firstName: reqString,
+    lastName: reqString,
+    userImage: reqString,
+    nickName: noReqString,
+    dateOfBirth: { type: Date, default: Date.now },
```

Comment:
I don't think having the present day as a default here makes much sense. Thoughts? :)

Reply:
ok, which year should i change the date to?

Reply:
I would suggest not to have any default for the `dateOfBirth` field.